documentation/milestones/individual_milestones/static_site_public_launch_infrastructure_content.md

```diff
@@ -0,0 +1,258 @@
+
+These content updates should be approved following the content review process outlined in our wiki.
+
+### Infrastructure improvements
+
+The focus of infrastructure improvements for this deliverable should be on changes that will help maintain the responsiveness and availability of the site as traffic increases. While the initial public announcement of the site will be targeted to an audience of around 1,000 users, we should should have a plan in place that allows us to easily support increases in site traffic.
+
+In particular, it would be helpful to review our approach to the following:
+
+- Employing static site generation (SSG) when possible
+- Caching the site via a content delivery network (CDN) or test load times to meet appropriate response times
+- Load balancing and/or proxying requests
+- Auto-scaling infrastructure resources based on volume of traffic
+- Other strategies as recommended by the engineering team
+
+### Incident response
+
+This deliverable also involves adopting or updating a set of systems and procedures to support with incident response in the event that the site experiences downtime.
+
+In particular, it would be helpful to review our approach to the following:
+
+- **Systems**
+  - Heartbeat monitoring of internal services and public urls to ensure that the website is live and publicly accessible
+  - Automated notifications when:
+    - services are down
+    - 4xx or 5xx requests reach a certain threshold
+    - resources reach a certain usage threshold or are auto-scaled
+  - Logging capabilities (e.g. search, filtering) and retention policies to assist with troubleshooting
+- **Procedures**
+  - Internally-defined service level agreements for availability and incident response
+  - Documented incident response plan with steps and roles identified
+  - Documented plan for receiving and responding to alerts
+  - Staff trainings, tabletop and/or dry-runs for common types of incidents
+
+### Communication strategy
+
+The two main communication outputs needed for this deliverable are:
+
+- An email campaign that is sent to the subset of the grants.gov mailing list with whom we plan to share the site.
+- A communications plan that outlines future plans for stakeholder engagement about the Simpler.Grants.gov initiative. This plan should answer the following questions:
+  - Which stakeholder groups are we planning to engage?
+  - Who is responsible for managing the relationship with each group?
+  - What is the primary channel and cadence for those communications?
+
+Both the initial email communication and the communication plan should be reviewed and approved by key stakeholders in the recurring communications meetings.
+
+### Email subscription mechanism
+
+One stretch goal for this deliverable is to adopt a service that allows site visitors to subscribe to a mailing list to receive updates about the Simpler.Grants.gov initiative. As part of this deliverable the team should answer the following questions:
```

Comment:
The email signup is part of the draft content we're circulating for review, so I don't think it's a stretch goal any more and is rather a core goal. 

Additional, the Figma prototypes include a survey tool for signing up to be a user research participant, so we should add that here as well.